provision/kubernetes/builder.go

```diff
@@ -77,6 +78,32 @@ func (c *KubeClient) ImageTagPushAndInspect(a provision.App, imageID, newImage s
 	return &inspectData.Image, inspectData.Procfile, &inspectData.TsuruYaml, nil
 }
 
+func (c *KubeClient) DownloadFromContainer(app provision.App, imageName string) (io.ReadCloser, error) {
+	client, err := clusterForPool(app.GetPool())
+	if err != nil {
+		return nil, err
+	}
+	reader, writer := io.Pipe()
+	stderr := &bytes.Buffer{}
+	go func() {
+		opts := execOpts{
+			client: client,
+			app:    app,
+			image:  imageName,
+			cmds:   []string{"cat /home/application/archive.tar.gz"},
+			stdout: writer,
+			stderr: stderr,
+		}
+		err := runIsolatedCmdPod(client, opts)
+		if err != nil {
+			writer.CloseWithError(err)
```

Comment:
`stderr` is never used, maybe it could be used here as in:
```suggestion
			writer.CloseWithError(errors.Wrapf(err, "error reading archive, stderr: %q", stderr.String()))
```